### DISCUSSION TITLE
German article about property graphs, include Property Graph Exchange Format

### DISCUSSION THREAD

**Discussion by nichtich** [pg-format/pg-formatter#45] [Show and tell]
I wrote a short article about property graphs for the German magazine of my employer. It's not proofread yet and will be published formally in May. The preprint is available at <https://jakobib.github.io/pgraphen2024/>.

**Comment by hchiba1** [comment]
Nice introduction to PG format!

**Comment by hchiba1** [comment]
> Rein formal bestehen Daten auch im RDF-Modell aus Knoten und Kanten, wobei Knoten als “Ressourcen” und Kanten als “Triples” bezeichnet werden.

> In purely formal terms, data also consists of nodes and edges in the RDF model, whereby nodes are referred to as "resources" and edges as "triples".

Maybe a minor point: a triple would be typically represented as node-edge-node.
(Or, consider the "edge" to include nodes at both ends..?)
